styles/tables/mod.css

```diff
@@ -1,44 +1,59 @@
+/* Modernized table styles */
+
 table {
-  display: block;
+  width: 100%;
   max-width: 100%;
   margin: 0 auto 1rem;
   border-collapse: collapse;
-  border-spacing: 0;
-  inline-size: fit-content;
   overflow-x: auto;
+  border-radius: 0.5rem;
```

Comment:
This has no effect:
![image](https://github.com/lowlighter/matcha/assets/22963968/76f2ccba-4df9-4963-930b-9a57d045d4cc)